pkg/apis/coordination/v1beta1/register.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Can someone clarify for me what belongs in pkg, and what belongs in staging?
E.g. if you look at the extensions group, the organization of which pieces are in which place is slightly different from here - maybe it's just superficial?

Reply:
Stuff that only kube-apiserver needs, such as register.go, should be in pkg. 

Staging is published out to other repos for use by clients.